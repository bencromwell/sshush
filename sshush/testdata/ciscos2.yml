---
ciscos:
  Config:
    Ciphers: aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    KexAlgorithms: +diffie-hellman-group1-sha1
    HostKeyAlgorithms: ssh-rsa,ssh-dss
    PubkeyAuthentication: "no"
  Hosts:
    - oldas*.adm
    - oldcs*.adm
    - cs1.office.adm
    - cs2.office.adm
    - ms1.office.adm
    - as1.office.adm
    - as2.office.adm
    - as3.office.adm
    - as4.office.adm
    - ps1.office.adm
    - ps2.office.adm

older_ciscos:
  Extends: ciscos
  Config:
    Ciphers: aes128-cbc,3des-cbc,aes192-cbc,aes256-cbc
  Hosts:
    - es*.office.adm
    - cr1.office2.adm
