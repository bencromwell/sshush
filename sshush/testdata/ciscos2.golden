# Generated by sshush v0.0.0-dev
# From testdata/ciscos2.yml

# ciscos
Host as1.office.adm
    HostName as1.office.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host as2.office.adm
    HostName as2.office.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host as3.office.adm
    HostName as3.office.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host as4.office.adm
    HostName as4.office.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host cs1.office.adm
    HostName cs1.office.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host cs2.office.adm
    HostName cs2.office.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host ms1.office.adm
    HostName ms1.office.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host oldas*.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host oldcs*.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host ps1.office.adm
    HostName ps1.office.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host ps2.office.adm
    HostName ps2.office.adm
    Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

# older_ciscos
Host cr1.office2.adm
    HostName cr1.office2.adm
    Ciphers aes128-cbc,3des-cbc,aes192-cbc,aes256-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no

Host es*.office.adm
    Ciphers aes128-cbc,3des-cbc,aes192-cbc,aes256-cbc
    HostKeyAlgorithms ssh-rsa,ssh-dss
    KexAlgorithms +diffie-hellman-group1-sha1
    PubkeyAuthentication no
